src/nvim/tui/input.c

```diff
@@ -49,15 +49,7 @@ void tinput_init(TermInput *input, Loop *loop)
   int curflags = termkey_get_canonflags(input->tk);
   termkey_set_canonflags(input->tk, curflags | TERMKEY_CANON_DELBS);
   // setup input handle
-#ifdef WIN32
-  uv_tty_init(&loop->uv, &input->tty_in, 0, 1);
-  uv_tty_set_mode(&input->tty_in, UV_TTY_MODE_RAW);
-  rstream_init_stream(&input->read_stream,
-                      (uv_stream_t *)&input->tty_in,
-                      0xfff);
```

Comment:
`tty_in` field is no longer used. I guess it could be removed from `struct term_input`